ij/src/main/kotlin/org/jetbrains/academy/test/system/ij/analyzer/IjCodeAnalyzerUtil.kt

```diff
@@ -79,3 +79,17 @@ fun PsiFile.findMethodsWithContent(content: String): List<String> =
         val methods = extractElementsOfTypes(KtNamedFunction::class.java)
         methods.filter { it.getBlockBody() == formattingContent }.mapNotNull { it.name }.toList()
     }
+
+/**
+ * Finds all the methods in the file where a specific method is called.
+ *
+ * @param methodName The name of the method to search for its usages.
+ * @return A list of strings containing the names of methods where the specified method is called.
+ */
+fun PsiFile.findMethodsWhereMethodIsCalled(methodName: String): List<String> =
```

Comment:
Maybe we can name it `findMethodUsages`? I'm not sure that the method name is enough for this search, I would also check the parameters list.
Also, I would first find all functions, and then analyze references inside them.

Reply:
I changed to check not only the name of the method, but the whole method call. 

If I first find all functions and then analyse the references inside them, it's not clear to me what to do in the case of nested functions, because the method call will be found in both methods.

Reply:
@anchouls There might be several methods with the same name but different list of parameters, that's why I suggest to check the list of parameters (they're count and types)